test/Graphics/ConvexShape.test.cpp

```diff
@@ -42,4 +43,31 @@ TEST_CASE("[Graphics] sf::ConvexShape")
         convex.setPoint(0, {3, 4});
         CHECK(convex.getPoint(0) == sf::Vector2f(3, 4));
     }
+
+    SUBCASE("Reasonable geometric center")
```

Comment:
This appears to be creating circles of varying size and testing that their centroid is correct. Can you reflect that in the `SUBCASE` name somehow?